### Exercise 2: Computing the percentage decrease in infant mortality rates of 8 European countries between 1860 and 2020

Here are the numbers we discussed in the text in a table. I named the variables for the infant mortality rate in this exercise MR-1860 for  the infant mortality rate in 1860 and MR-2020 for the corresponding mortality rate in 2020.

|Country |Continent | MR-1860| MR-2020|
|:-------|:---------|-------:|-------:|
|Austria |Europe    |   0.237|  0.0030|
|Belgium |Europe    |   0.139|  0.0034|
|Denmark |Europe    |   0.136|  0.0031|
|France  |Europe    |   0.150|  0.0034|
|Germany |Europe    |   0.260|  0.0031|
|Norway  |Europe    |   0.102|  0.0018|
|Spain   |Europe    |   0.174|  0.0027|
|Sweden  |Europe    |   0.124|  0.0021|

In case you wonder how I did this table in the Jupyter notebook, look at the cell input and study the code. For
textcells in Jupyter you can use a markup language called Markdown. Mardown will allow you to format text by using certain markup symbols that the Jupyter notebook can interpret and process. Here is a link to a cheat sheet summarizing the most important markup instructions: https://www.markdownguide.org/cheat-sheet/

I encourage you as facilitators to familiarize yourself with markup and teach it to the students you work with when working with the Jupyter Notebooks.

The infant mortality rate in Austria was $0.237$ in 1860 and was $0.0030$ 160 years later in 2020. The percentage decrease over this period is computed as:
\begin{equation*}
\frac{0.237 - 0.003}{0.237}
\end{equation*}

you can compute this by hand or by calculator or by R. Let me use R for the sake of the example

In [1]:
((0.237 - 0.003)/0.237)*100

[1] 98.73418

This is a reduction of roughly 99 % percent. One could also say that the mortality rate decreased by a factor of 78. If you express the change as a relative change in percent you see how enormous the reduction was over a period of 160 years. This is next to a miracle, after centuries of extremely high rates.

In case you are uncertain how to compute percentage changes, let us discuss some examples: 

Example 1: Suppose you have worked in a Supermarket for a wage of 10 (in some monetary unit) and your pay is increased to 12. What is your pay increase in percent? The way to compute the percentage increase is:
\begin{equation*}
\frac{(12-10)}{10}
\end{equation*}

which is $0.20$. Take this times 100 and you have the increase in percent, which is $20$ %.

Example 2: The staff of a company decreased from 40 to 29. What was the percentage decrease in its labor force?
\begin{equation*}
\frac{(29 - 40)}{29}
\end{equation*}

which is $0.275$. Take this by 100 and you have  $27.5$ %.

Did you see the general pattern. When computing a percentage change, the formula you apply is

\begin{equation*}
\frac{(final - initial)}{initial} \times 100
\end{equation*}

This is the formula for a percentage change. When this change is positive, we say naturally that we have an increase, when it is negative we say we have a decrease and fold the minus sign into the workd decrease. So in the second example, we would say that the labor force of the company decreased by$27.5$ %.

When the percentage change is 100 it means that something doubled, when it is 1000 it means that something has increased 10 fold, or as in the case of infant mortality in Austria between 1860 and 2020 has decreased 78 fold. 

We can also do the computations with R based on what we have learned in the course so far. Let's first store the values in two objects, which we call 'mr_1860' and 'mr_2020'.

In [2]:
mr_1860  <- c(0.237, 0.139, 0.136, 0.15, 0.26, 0.102, 0.174, 0.124)
mr_2020  <- c(0.003, 0.0034, 0.0031, 0.0034, 0.0031, 0.0018, 0.0027, 0.0021)

Now we use the fact that R can compute component wise and apply the formula for the percentage change and round to 2 digits:

In [3]:
round(((mr_1860 - mr_2020)/mr_1860)*100, 2)

[1] 98.73 97.55 97.72 97.73 98.81 98.24 98.45 98.31

So this means we have:

|Country |Percentage decrease in infant mortality from 1860 - 2020 |
|:-------|:-------------------------------------------------------:|
|Austria |98.73 %                                                  |
|Belgium |97.55 %                                                  | 
|Denmark |97.72 %                                                  |   
|France  |97.73 %                                                  |   
|Germany |98.81 %                                                  |   
|Norway  |98.24 %                                                  |   
|Spain   |98.45 %                                                  |   
|Sweden  |98.31 %                                                  |

Usually you dp not encounter percentage number like this, because the difference in the mortality rates are so enormous. Suppose you were a bit unsure whether this could be indeed correct. Here is a way to check your calculation. A decrease by, for example of 98.73 %, is a decrease by 0.9873 in decimal notation. So if your rate was initially 0.237 and you subtract $0.237 \times 0.9873$ you get 0.0030099 as you should if rounded to the last four digits.